src/signals/incident-management/containers/IncidentDetail/components/SplitNotificationBar/index.js

```diff
@@ -17,43 +52,47 @@ const getErrorMessage = (status) => {
   }
 };
 
+const getDelimiter = (key, length) => {
+  if (key === length - 2) {
+    return ' en ';
+  }
+  return key < length - 1 ? ', ' : '';
+};
+
+const renderCloseButton = (onDismissSplitNotification) => (
+  <StyledCloseButton
+    size={20}
+    variant="blank"
+    icon={<Close />}
+    data-testid="splitNotificationBarCloseButton"
+    onClick={onDismissSplitNotification}
+  />
+);
+
 const SplitNotificationBar = ({ data, onDismissSplitNotification }) => (
-  <div data-testid="split-notification-bar">
+  <StyledWrapper data-testid="splitNotificationBar">
     {data && data.id && data.created && data.created.children && Array.isArray(data.created.children) ?
-      <div className="split-notification-bar success">
-        <div className="split-notification-bar__body">
-          <button
-            className="split-notification-bar__close-button"
-            type="button"
-            data-testid="split-notification-bar-close-button"
-            onClick={onDismissSplitNotification}
-          />
-
-          Melding {data.id} is gesplitst in
-          {data.created.children.map((item) =>
-            (<span key={item.id} className="split-notification-bar__link">
-              &nbsp;<NavLink to={`/manage/incident/${item.id}`}>{item.id}</NavLink>
-            </span>)
-          )}
-        </div>
-      </div>
+      <StyledSuccess>
+        {renderCloseButton(onDismissSplitNotification)}
+
+        Melding {data.id} is gesplitst in&nbsp;
+        {data.created.children.map((item, key) =>
+        (<span key={item.id}>
+          <NavLink to={`/manage/incident/${item.id}`}>{item.id}</NavLink>
+          {getDelimiter(key, data.created.children.length)}
```

Comment:
Why not 
```jsx
data.created.children.map((item, key) => (
  <span key={item.id}>
    <NavLink to={`/manage/incident/${item.id}`}>{item.id}</NavLink>
  </span>
)).join(' en ');
```

Reply:
hij moet twee varianten ondesteunen: 
- "24 en 25"
- "23, 24 en 25"